lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -305,6 +305,20 @@ namespace Js
                 PropertyString * propertyString = (PropertyString *)propName;
                 *propertyRecord = propertyString->GetPropertyRecord();
             }
+            else if (VirtualTableInfo<Js::LiteralStringWithPropertyStringPtr>::HasVirtualTable(propName))
```

Comment:
>VirtualTableInfo<Js::LiteralStringWithPropertyStringPtr>::HasVirtualTable(propName [](start = 21, length = 82)

Would it make sense to add equivalent code at other places where we check for PropertyStrings? For example, ForInObjectEnumerator::MoveAndGetNext, and TMapKey_ConvertKey and TMapKey_ConvertKey_TTD in SimpleDictionaryTypeHandler.cpp.